jwt-auth/2.1/rest-client/README.adoc

```diff
@@ -0,0 +1,31 @@
+---
+title: "MicroProfile Rest Client 3.1"
+date: 2024-05-15
```

Comment:
Hi @Emily-Jiang how did you arrive at this date?

Everything looks good. We are just looking at the [release timeline](https://docs.google.com/spreadsheets/d/1yYI5pdRCoOT5GaZ6KMNTfg3mQ5Hwxxv45SW5WBsZAmY/edit#gid=31204409) and can't match it up to any target.

Reply:
I delayed this date a bit. Let me update to sync up with the plan to avoid confusion.